From e1a95254dd0ec874f884160aaf0f5ce7947a06c8 Mon Sep 17 00:00:00 2001
From: Himal Ghimiray <himalg@codeaurora.org>
Date: Thu, 12 Oct 2017 17:55:21 +0530
Subject: Adding nosuid flag for partition mounting in apq8053.

-Adding flag for mounting of persist, data, dsp and cache.

Change-Id: I7f7ae176c83e51ea662885862a742d1ca6841c0d
---
 recipes-bsp/startup-scripts/files/apq8053/find_partitions.sh | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/recipes-bsp/startup-scripts/files/apq8053/find_partitions.sh b/recipes-bsp/startup-scripts/files/apq8053/find_partitions.sh
index 8094055..92ca56f 100755
--- a/recipes-bsp/startup-scripts/files/apq8053/find_partitions.sh
+++ b/recipes-bsp/startup-scripts/files/apq8053/find_partitions.sh
@@ -39,9 +39,10 @@ FindAndMountEXT4 () {
    /sbin/restorecon -R $2
 }
 
-FindAndMountEXT4 userdata /data relatime,data=ordered,noauto_da_alloc,discard,nodev
-FindAndMountEXT4 dsp /dsp relatime,data=ordered,noauto_da_alloc,discard,ro,noexec,nodev
-FindAndMountEXT4 persist /persist relatime,data=ordered,noauto_da_alloc,discard,noexec,nodev
-FindAndMountEXT4 cache  /cache relatime,data=ordered,noauto_da_alloc,discard,noexec,nodev
+
+FindAndMountEXT4 userdata /data relatime,data=ordered,noauto_da_alloc,discard,nodev,nosuid
+FindAndMountEXT4 dsp /dsp relatime,data=ordered,noauto_da_alloc,discard,ro,noexec,nodev,nosuid
+FindAndMountEXT4 persist /persist relatime,data=ordered,noauto_da_alloc,discard,noexec,nodev,nosuid
+FindAndMountEXT4 cache  /cache relatime,data=ordered,noauto_da_alloc,discard,noexec,nodev,nosuid
 
 exit 0
-- 
cgit v1.1

